src/app/shared/components/users-billing-info/users-billing-info.component.html

```diff
@@ -125,10 +126,17 @@ <h3 class="form-step-card-title text-dark mb-3">
                         <strong>{{upgradeAmount|currency:currency}}</strong>
                       </div>
                     </div>
-
+                  </div>
                 </div>
               </div>
-              <div class="form-step-card" *ngIf="paymentMethod===paymentMethodType.BITCOIN && planType!==planTypeEnum.FREE">
+              <div class="monero"*ngIf="Monero">
+                <h1 class="moneroInfo">Payment Information</h1>
+                <p class="moneroInfo">To complete your payment using Monero (XMR) please create a free account then email
+                  Support@ctemplar.com. Our Support team will send you instructions on how to complete the payment
```

Comment:
- I think our support mail address should be like this `support@ctemplar.com`
- for the class attribute, same issue with above